docs/source/kedro_project_setup/settings.md

```diff
@@ -3,7 +3,7 @@
 A Kedro project's `settings.py` file contains the application settings for the project, including registration of Hooks and library components. This page explains how settings work, and which settings are available.
 
 ```{note}
-Application settings is distinct from [run time configuration](configuration.md), which is stored in the `conf` folder and can vary by configuration environment, and [pyproject.toml](../faq/architecture_overview.md#kedro-project) , which provides project metadata and build configuration.
+Application settings is distinct from [run time configuration](configuration.md), which is stored in the `conf` folder and can vary by configuration environment, and according to the contents of `pyproject.toml`, which provides project metadata such as `package_name`, `project_name` and `project_version` (the version of Kedro with which the project was generated).
```

Comment:
See above

Reply:
Ditto. I added this bit of text from the architecture doc as I removed it. It doesn't mention the `context_path` though, which is what you were referring to previously. I think I may need a bit of help with this (adding more docs about `pyproject.toml`)